test/compilable/test21861.d

```diff
@@ -0,0 +1,33 @@
+// https://issues.dlang.org/show_bug.cgi?id=21861
```

Comment:
Test should also check for something like `@("S"): <multiple declarations>`

Reply:
Any tests for this? Can something break?

Reply:
@BorisCarvajal please add a test with a UDA declaration as suggested by @MoonlightSentinel and we are good to go.

Reply:
Can't test that. As I said before:
> Currently, at function scope the parser only allows one declaration after UDAs.

Both `attrib { decls... }` and `attrib: decls...` are not allowed at statement level.

Reply:
I've just recalled already putting those tests for future parser changes to trigger an assert.

```d
    static assert(!__traits(compiles, mixin("{ @(1) { int x; int y; } }")));
    static assert(!__traits(compiles, mixin("{ @(1): int x; int y; }")));
```